conf/main/grakn.properties

```diff
@@ -109,6 +109,7 @@ loader.threads=0
 #Authentication
 password.protected=false
 JWT.secret=Anzdz9l4TG8N5y18oaTDueqJQca1aX7loaz0U8Hz
+admin.password=grakn
```

Comment:
Yes, that's an idea. And same would apply to the JWT.secret above, which is also in plain text. Passwords are not easy to manage - there are enterprise products just for that, especially since they proliferate for all sorts of components and it's hard to keep track of them. 

The problem with storing in system graph is that if you forget/lose the password, you can't login anymore at all and you have no means of recovery. If that one and only that one password is externalized, at least you can reset it somewhere and recover. The proper method would be probably to have some "security provider" that gives the admin password so a user can pull it from a secure external place.